### PR TITLE
dev-robertoalx-v3-about-me

### PR DESCRIPTION
**Pull request by robertoalx**

### CODE REVIEW

**Review by marianzburlea — APPROVED** [review]
💯

**Review by blowymind99 — APPROVED** [review]
💯

File: src/_data/about.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "sectionHeader": "Welcome to my Sanctuar",
```
Comment (QuintyHH) [inline src/_data/about.json]: Misspelled 'Sanctuary'

File: src/style.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+@import '_component/base/base';
+@import '_component/base/source-code-pro-font';
+@import '_component/about/about';
+@import '_asset/font/style'
```
Comment (QuintyHH) [inline src/style.scss]: Please add a newline

File: src/_component/base/source-code-pro-font.scss [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+}
+/* latin */
+@font-face {
+  font-family: 'Source Code Pro';
+  font-style: normal;
+  font-weight: 900;
+  font-display: swap;
+  src: url(https://fonts.gstatic.com/s/sourcecodepro/v13/HI_XiYsKILxRpg3hIP6sJ7fM7PqtvszDs-cq.woff2) format('woff2');
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
```
Comment (QuintyHH) [inline src/_component/base/source-code-pro-font.scss]: Please add a newline

**Review by QuintyHH — CHANGES_REQUESTED** [review]

**Review by adelacecilia — APPROVED** [review]
Good job, Rob! 👍 💯

**Review by psyhedeliq — APPROVED** [review]
🤘

**Review by BogdanNegura — APPROVED** [review]
👍

File: src/_component/about/about.scss [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    font-size: 24px;
+  }
+
+  &main-title,
+  &sub-title {
+    color: #707070;
+    text-align: center;
+  }
+
+  &main-title{
```
Comment (RalucaRbc) [inline src/_component/about/about.scss]: Just for readability, you may need an empty space between the class name and `{`; here and some missing empty spaces above 😄

File: src/_component/about/about.pug [hunk 5]
```diff
@@ -0,0 +1,33 @@
+.column(span="row")
+  .section__header-wrapper
+    .section__header=data['about'].sectionHeader
```
Comment (RalucaRbc) [inline src/_component/about/about.pug:3]: Nothing wrong with your  `data['about'].sectionHeader` but you could have had just `data.about.sectionHeader` as you don't have any special character like `-` in the name of your file about.json

**Review by RalucaRbc — APPROVED** [review]
Well done overall Robert 🚀

**Review by marianzburlea — APPROVED** [review]
💯

**Review by biggicode — APPROVED** [review]

File: src/_asset/font/style.scss [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+.my-cv__arrow-down {
+  &:before {
+    content: $my-cv__arrow-down; 
+  }
+}
+.my-cv__arrow-up {
+  &:before {
+    content: $my-cv__arrow-up; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/style.scss:38]: Empty line needed for readability

File: src/_asset/font/style.scss [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+.my-cv__book {
+  &:before {
+    content: $my-cv__book; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/style.scss:43]: Empty line needed for readability

File: src/_asset/font/style.scss [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+.my-cv__codetap {
+  &:before {
+    content: $my-cv__codetap; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/style.scss:48]: Empty line needed for readability

File: src/_asset/font/style.scss [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+.my-cv__codewars {
+  &:before {
+    content: $my-cv__codewars; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/style.scss:53]: Empty line needed for readability

File: src/_asset/font/style.scss [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+.my-cv__commits {
+  &:before {
+    content: $my-cv__commits; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/style.scss:58]: And so on

**Review by MariaTeo — APPROVED** [review]
Nice work overall

**Review by marianzburlea — APPROVED** [review]

**Pull request merged**